Makefile

```diff
@@ -26,7 +26,9 @@ verify: ## Verify the code using various linters
 	ruff check scripts --per-file-ignores=scripts/*:S101
 
 update-ocp-docs: ## Update the plaintext OCP docs in ocp-product-docs-plaintext/
-	scripts/get_ocp_plaintext_docs.sh 4.15
+	@for OCP_VERSION in 4.15 4.16; do \
```

Comment:
same comment as above...would be slightly less effort to maintain if we add another version if this picked up the list of versions by just listing what's in the directory and then iterating each subdir.